### PR TITLE
Improve events parser

### PR DESCRIPTION
**Pull request by leonardehrenfried**
This PR makes the event parser more complete by adding more examples and fixing a few fields.

It also contains the following:
 - Separation between`Source` and `SourceInput` to reflect better what they actually are. (`Source` was too overloaded).
- Adds a source type `Account`
- Move event json example files into their own account

### CODE REVIEW

File: src/main/scala/org/mdedetrich/stripe/v1/Charges.scala [hunk 1]
```diff
@@ -168,23 +169,31 @@ object Charges extends LazyLogging {
...
-      addressCountry: Option[String],
-      addressLine1: Option[String],
-      addressLine2: Option[String],
-      name: Option[String],
-      addressState: Option[String],
-      addressZip: Option[String]
-  ) extends MaskedCardSource
-
-  implicit val maskedSourceReads: Reads[MaskedSource] = (
+  sealed trait Source
```
Comment (mdedetrich) [inline src/main/scala/org/mdedetrich/stripe/v1/Charges.scala]: Can you make this a `sealed abstract class`? It just has better binary compatibility compared to `sealed trait`

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Just need to change the sealed trait to sealed abstract class

**Comment by leonardehrenfried** [comment]
I've changed it to a sealed abstract class.

Once you've merged, could you release a 0.1.3?

**Comment by mdedetrich** [comment]
Don't have laptop on me now, will release tomorrow

**Pull request merged**